mods/ra/rules/structures.yaml

```diff
@@ -56,17 +58,27 @@ MSLO:
 	CanPowerDown:
 		PowerupSound: EnablePower
 		PowerdownSound: DisablePower
-	RequiresPower:
+		PowerdownCondition: powerdown
+	GrantConditionOnPowerState@LOWPOWER:
+		Condition: lowpower
 	SupportPowerChargeBar:
 	Power:
+		RequiresCondition: !disabled
 		Amount: -150
 	MustBeDestroyed:
 		RequiredForShortGame: false
 	WithNukeLaunchAnimation:
+	WithDecoration@POWERDOWN:
```

Comment:
These should override the `ReferencePoint` to center the decoration:

![screen shot 2017-09-15 at 20 00 31](https://user-images.githubusercontent.com/167819/30499020-cddc93de-9a50-11e7-9613-8980a465fbc0.png)

I actually prefer it not zooming, but this does make it inconsistent with the repair glyph.  If we keep this behaviour then we'll probably need to block on porting that to `WithDecoration` too.

Reply:
> plus some kind of cycling condition trait so that only one decoration is shown at once

I don't agree with this part. Apart from sounding a bit too complicated for this, I highly doubt switching back and forth between the two would look good with the available art.

In my opinion we should just tweak the sequence offset(s) so they don't overlap, that way we don't need to scope-creep, don't need to deal with getting the transition right, and don't need to deal with players that would prefer to have both indicators visible all the time.

Reply:
Ok, that works too.

Reply:
The blinking can also be implemented at the `WithDecoration` trait level: `Created` can query and cache any other traits that declare the same position, and then `Tick` can cycle the visibility between these in turn.  If you want I could do a prototype demonstrating this.

Reply:
Fixed the reference points and disabled zooming on the RepairIndicator.